packages/react/src/auto/polaris/submit/PolarisSubmitResultBanner.tsx

```diff
@@ -0,0 +1,46 @@
+import { Banner, BannerProps } from "@shopify/polaris";
+import React, { useEffect, useState } from "react";
+import { useAutoFormMetadata } from "../../AutoFormContext.js";
+
+export const PolarisSubmitResultBanner = (props: { successBannerProps?: BannerProps; errorBannerProps?: BannerProps }) => {
```

Comment:
What do you think about `propsOnError` and `propsOnSuccess`? Or even like this:
```
props: {
  bannerProps: (isSuccess: boolean): BannerProps
}
```
Then when users want to modify it based on the state, they can do something like this:
```jsx
<PolarisSubmitResultBanner
  bannerProps={(isSuccess) => {
    if(isSuccess) {
      ...
    } else {
      ...
    }
  }}
/>
```

Reply:
In my opinion, I'd prefer to keep the props as separate objects instead of a boolean controlled one

I feel like a user looking through the suggested props will see something like `bannerProps` and be confused when it is a function instead of an object